.ci/install_runtime.sh

```diff
@@ -65,7 +65,10 @@ if [ "$KATA_HYPERVISOR" = "nemu" ]; then
 fi
 
 if [ "$KATA_HYPERVISOR" = "qemu" ] && [ "$experimental_qemu" == "true" ]; then
+	runtime_repo="github.com/kata-containers/runtime"
+
 	echo "Enable qemu virtiofs configuration.toml"
+	sudo mv "$GOPATH/src/${runtime_repo}/cli/config/configuration-qemu-virtiofs.toml" "${PKGDEFAULTSDIR}"
```

Comment:
I think this should be:

```
sudo mv "${PKGDEFAULTSDIR}/configuration-qemu-virtiofs.toml" "${PKGDEFAULTSDIR}/configuration.toml"
```
and remove the below one.

Reply:
@chavafg the problem with that is that `configuration-qemu-virtiofs.toml` is not found at `${PKGDEFAULTSDIR}`

Reply:
it should be there after running `make install`, which the scripts run before these lines, I tried locally and I see the file there:
```
$ ls -l /usr/share/defaults/kata-containers
total 108
-rw-r--r-- 1 root root  9954 Oct  7 15:46 configuration-acrn.toml
-rw-r--r-- 1 root root 14305 Oct  7 15:46 configuration-fc.toml
-rw-r--r-- 1 root root 16672 Oct  7 15:46 configuration-nemu.toml
-rw-r--r-- 1 root root 17201 Oct  7 15:46 configuration-qemu-virtiofs.toml
-rw-r--r-- 1 root root 17221 Oct  7 15:46 configuration-qemu.toml
-rw-r--r-- 1 root root 17166 Oct  7 15:46 configuration.toml
```

Reply:
@chavafg, I also tried locally and fails because is saying that `configuration-qemu-virtiofs.toml` is missing at that path, I ran the `.ci/setup.sh`